### ISSUE TITLE
Update PMP status element

### ISSUE THREAD

**Issue by daniel-tabarcea** [unicef/etools-issues#398]
Make the status component reusable and update it to new specs

**Comment by evrimm** [comment]
@daniel-tabarcea Can you take a look at #378 and incorporate this change as well into the reusable element?

**Comment by daniel-tabarcea** [comment]
Ok, I'll make it look as requested in #378

**Comment by adi130987** [comment]
@evrimm regarding status indicator, do we need to display all the possible statuses an item(agreement/pd) has?

For example for a PD that is completed(ended) we show all the statuses, but the PD might never been in suspended status, but we will show it. Shouldn't we keep a status history and show only those statuses completed, if the PD never been in suspended status this will be in inactive state.

**Comment by evrimm** [comment]
@adi130987  so sorry, I am only seeing this comment now. I have github notification issues that I'm trying to resolve. 

By the way I split 378 into 2 cards, one for Trips, one for PMP. Now the PMP-related card is #418. I am adding screens to address your question, if I have understood your question right... Basically, no need to show status history (i.e if an agreement or PD has been Suspended and then Unsuspended, no need to show that at one point in time it was suspended)

**Comment by adi130987** [comment]
@evrimm this means that not all possible statuses an item has will be displayed(just like current functionality is right now). For example in #418 screenshots for an agreement the suspended and terminated state are on the same position and can not be shown in the same time.

Also the the icons for suspended and terminated are different. Should these icons be replaced as requested here: "Expected behavior: Based on user feedback, the current status indicator icon should look the same as the past statuses, specifically the background is green and checkmark is displayed."?

**Comment by evrimm** [comment]
@adi130987 correct, it will be like the current functionality, not all possible statuses an item has will be displayed. 

Second one is a good point, the icons or suspended and terminated should be different as they are exceptional states and not completes states per se. I will add this to card #418 so that it's clear.

**Comment by roxanavlad** [comment]
test result: pass
tested on dev environment on Google Chrome and Internet Explorer

**Comment by mpawlowski85** [comment]
@evrimm can you please test it?

**Comment by evrimm** [comment]
passed
